src/script/lua_api/l_util.cpp

```diff
@@ -459,5 +489,8 @@ void ModApiUtil::InitializeAsync(AsyncEngine& engine)
 
 	ASYNC_API_FCT(mkdir);
 	ASYNC_API_FCT(get_dir_list);
+
+	ASYNC_API_FCT(encode_base64);
```

Comment:
If this is going to be an async api, it needs to be documented menu_lua_api.txt as well as lua_api.txt.